docs/guides/overriding-a-react-context.md

```diff
@@ -0,0 +1,196 @@
+---
+order: 30
+label: Overriding a React context
+---
+
+# Overriding a React context
+
+In a federated application using [Module Federation](https://webpack.js.org/concepts/module-federation/), it's typical to configure various global [React contexts](https://legacy.reactjs.org/docs/context.html) at the root of the host application. These contexts are usually consumed down the line by the layouts and pages of the remote modules.
+
+Let's take a simple example using a `BackgroundColorContext`:
+
+```tsx !#16-21 host/src/App.tsx
+import { useAppRouter } from "@sample/shell";
+import { BackgroundColorContext } from "@sample/shared";
+import { useAreModulesReady } from "@squide/webpack-module-federation";
+import { RouterProvider } from "react-router-dom";
+
+export function App() {
+    const isReady = useAreModulesReady();
+
```

Comment:
extra line break for no reason?

Reply:
Oh I almost always had one between my hooks, all the samples of Squide has this line break!